.github/workflows/pr.yaml

```diff
@@ -48,10 +48,9 @@ jobs:
         timeout-minutes: 5
         run: |
           make deploy-konk-service
-      - name: Deploy konk-service-operator
+      - name: Deploy konk-service
```

Comment:
Can you make this `Deploy example-konk-service`, so that it is distinct from the previous name?